pages/login.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div
+    id="login"
+    class="flex flex-grow container mx-auto p-3 justify-center items-start"
+  >
+    <log-in-card @signUpClick="$router.push('/signup')" />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import LogInCard from '../components/LogInCard.vue'
+
+export default Vue.extend({
+  name: 'Login',
+
+  components: {
+    LogInCard,
+  },
```

Comment:
Does Nuxt require us to include the components used in page? Or are we able to omit them, so long as we use the autopathing that it offers? I can go either way on this, but if they're not required, I'd rather omit them, unless it's a case where a sub component help makes up the larger component, like in the `<NavLinks />`.

Reply:
I forgot about the autopathing. I'll remove that.